test/models/concerns/pxe_loader_suggestion_test.rb

```diff
@@ -20,6 +20,33 @@ def setup
     end
   end
 
+  context 'host with os param' do
+    def setup
+      @os = FactoryBot.create(:operatingsystem)
+      @os.os_parameters.create!(:name => "pxe-loader", :value => "PXELinux A")
+      @host = FactoryBot.create(:host, operatingsystem: @os)
+    end
+
+    test 'host suggests default PXEloader for OS' do
+      @host.suggest_default_pxe_loader
+      assert_equal 'PXELinux A', @host.pxe_loader
+    end
+  end
+
+  context 'host with model param' do
```

Comment:
how is this different to the test above?

Reply:
The context is different: `@host = FactoryBot.create(:host, operatingsystem: @os)` vs `@host = FactoryBot.create(:host)`